server/buildbuddy_server/buildbuddy_server.go

```diff
@@ -471,11 +471,7 @@ func (s *BuildBuddyServer) GetApiKeys(ctx context.Context, req *akpb.GetApiKeysR
 	if userDB == nil {
 		return nil, status.UnimplementedError("Not Implemented")
 	}
-	groupID := req.GetGroupId()
-	if err := perms.AuthorizeGroupAccess(ctx, s.env, groupID); err != nil {
-		return nil, err
-	}
-	tableKeys, err := userDB.GetAPIKeys(ctx, groupID)
+	tableKeys, err := userDB.GetAPIKeys(ctx, req.GetGroupId())
```

Comment:
Am I missing or something or does userdb.GetAPIKeys not validate that the user has access to the specified group?